django_project/core/settings/contrib.py

```diff
@@ -38,6 +39,13 @@
 
 CRISPY_TEMPLATE_PACK = 'bootstrap3'
 
+# Stripe Config
+STRIPE_KEY = "sk_test_OHW7bvLJhDtm1k2pI8AwIiEY"
```

Comment:
this stuff should come from environment variables rather so that we do not need to store sensitive data in git....